### DISCUSSION TITLE
The Github app crashes when looking at trending repositories.

### DISCUSSION THREAD

**Discussion by bnjmnjrk** [community/community#81704] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.142.1 (118013759)
Device: iPhone 7 (iOS 15.8)
</details>

**Comment by esahmed21** [comment]
Hi @bnjmnjrk 👋 

Sorry to hear that you encounter this bug. To help us figure it out, would you mind sharing more details about the steps to reproduce it, do you happen to have some filters applied in the trending repositories screen when the crash happens?

Thank you for reaching out to us!

**Reply by bnjmnjrk** [reply]
Thank you for your reply.

When i select the filter:
- time: this month
- programming language: rust
The app crashes. 
Every other combination seems to work.

**Reply by bnjmnjrk** [reply]
https://github.com/community/community/assets/106487517/16140007-69b2-4c89-960c-f9a70d0d9535

**Reply by esahmed21** [reply]
Hi @bnjmnjrk 👋 

Thank you for sharing these details! Your input has been helpful in pinpointing the issue, and our team is actively working on a fix.

We appreciate the time you took to provide this feedback. If you have any more insights or questions, feel free to share them with us.

Thanks again for your contribution!
